cats/__init__.py

```diff
@@ -48,24 +54,41 @@ def parse_arguments():
        cats -d 1 --loc RG1 --scheduler=at --command='ls'
     """
 
-    example_text = """
-    Examples\n
-    ********\n
+    config_text = indent_lines(
+        Path(__file__).with_name("config.yml").read_text(), spaces=8
+    )
+    example_text = f"""
+    Examples
+    ********
 
-    Cats can be used to report information on the best time to run a calculation and the amount
-    of CO2. Information about a 90 minute calculation in centeral Oxford can be found by running:
+    CATS can be used to report information on the best time to run a calculation
+    and the amount of CO2. Information about a 90 minute calculation in centeral
+    Oxford can be found by running:
 
-        cats -d 90 --loc OX1 --jobinfo="cpus=2,gpus=0,memory=8,partition=CPU_partition"
+        cats -d 90 --loc OX1
 
-    The `at` scheduler is available from the command line on  most Linux and MacOS computers,
-    and can be the easest way to use cats to minimise the carbon intensity of calculations on
-    smaller computers. For example, the above calculation can be scheduled by running:
+    The `at` scheduler is available from the command line on most Linux and
+    MacOS computers, and can be the easest way to use cats to minimise the
+    carbon intensity of calculations on smaller computers. For example, the
+    above calculation can be scheduled by running:
 
         cats -d 90 --loc OX1 -s at -c 'mycommand'
+
+    To report carbon footprint, pass the `--config` option to select a
+    configuration file and the `--profile` option to select a profile. An
+    example config file is given below:
+
+{config_text}
```

Comment:
Very neat. Is that some sort of sphinx include syntax?

Reply:
Just the standard f-string with the file read in